src/views/Eat.vue

```diff
@@ -1,26 +1,60 @@
 <script setup lang="ts">
 import { storeToRefs } from 'pinia';
+import { computed } from 'vue';
 import Content from '@/components/Content.vue';
 import PizzaCard from '@/components/PizzaCard.vue';
 import { usePizzaStore } from '@/stores/pizza.store';
+import { frenchFormatFromDate } from '@/utils';
 
 const pizzaStore = usePizzaStore();
-const { pizzaList } = storeToRefs(pizzaStore);
+const { timeslotList, pizzaList } = storeToRefs(pizzaStore);
 
-const { fetchAllPizzas } = pizzaStore;
+const { fetchAllPizzas, fetchNextTimeslots } = pizzaStore;
+
+const sortedTimeslotList = computed(() => Object.values(timeslotList.value).sort((pizza1, pizza2) => {
+  const time1 = new Date(pizza1.delivery_time).getTime();
+  const time2 = new Date(pizza2.delivery_time).getTime();
+  return time1 - time2;
+}));
+
+const pizzas = computed(() => {
+  if (sortedTimeslotList.value.length === 0) {
+    return pizzaList.value;
+  }
+
+  return Object.values(pizzaList.value)
+    .filter((pizza) => sortedTimeslotList.value[0].pizza.includes(pizza.id));
+});
 
 await fetchAllPizzas();
+await fetchNextTimeslots();
 </script>
 
 <template>
-  <Content name="Restauration"/>
+  <h1 class="title y-2 text-white">
+    Restauration
+  </h1>
+  <Content class="mb-8 flex flex-col gap-8 lg:px-24 [&_p]:indent-12" name="Restauration"/>
+  <section v-if="sortedTimeslotList.length" class="mx-4 mb-8 lg:px-24">
+    <h1 class="title mb-8 text-white">
+      Prochains créneaux
+    </h1>
+    <p>
+      Les prochains créneaux sont :
+    </p>
+    <ul class="list-disc">
+      <li v-for="timeslot in sortedTimeslotList" :key="timeslot.id" class="ml-16">
+        Créneau {{ frenchFormatFromDate(new Date(timeslot.delivery_time)) }}
+      </li>
+    </ul>
+  </section>
   <section>
-    <div class="title my-2 text-white">
-      Pizza
-    </div>
-    <div class="mb-12 grid w-full gap-4 px-4 md:grid-cols-2 xl:grid-cols-4">
+    <h1 class="title mb-8 text-white">
+      Pizzas {{ Object.values(timeslotList).length ? 'du Prochain Créneau' : '' }}
```

Comment:
Small performance detail (doesn't change much but still interesting to know): Object.values(...).length is slower than Object.key(...).length. This is due to the implementation of the disctionary in js, which is based on hashes.
To obtain the list of values, you have to first get the list of keys.

Also, instead of computing the values, you can use `sortedTimeslotList` to get the length which is already computed.